components/index.scss

```diff
@@ -228,12 +228,14 @@
     }
   }
 
-  .controlLabelError,
-  .controlLabelMediumSize {
-    align-self: flex-start; // otherwise would be centered between Input Field and error
-    display: flex;
-    align-items: center;
-    min-height: 46px; // height of input
```

Comment:
No we don't generally add numbers like this into variables unless it will be used in a lot of places. But this height is a little more specific to this component. Also, I am not restyling the component :D Just fixing a bug that I noticed on mobile.